app.py

```diff
@@ -227,7 +227,13 @@ def index(category=[]):
 def tag_index(slug):
     api_url = ''.join([INSIGHTS_URL, '/tags?slug=', slug])
     response = _get_from_cache(api_url)
-    tag = json.loads(response.text)[0]
+    if json.loads(response.text):
+        tag = json.loads(response.text)[0]
```

Comment:
This would cause the json to load twice. Could you please put the response in a variable before the if statement.